core/modules/comment/comment.module

```diff
@@ -2150,6 +2149,9 @@ function comment_form_submit($form, &$form_state) {
         backdrop_set_message(t('Your comment has been queued for review by site administrators and will be published after approval.'));
       }
     }
+    elseif (isset($comment->new)) {
```

Comment:
Suggest using the method `isNew()` instead. The property `new` is used for a different purpose in comments. See my comment in the main issue.

```suggestion
    elseif (!$comment->isNew()) {
```